cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -261,9 +215,15 @@ function apply_rook_op_img(){
 ########
 create_directories
 generate_operator_yaml "$@"
-generate_role_yaml
-generate_role_binding_yaml
-generate_service_account_yaml
+
+# Do not include Pod Security Policy (PSP) resources for CSV generation since OLM uses
+# Security Context Constraints (SCC).
+export DO_NOT_INCLUDE_POD_SECURITY_POLICY_RESOURCES=true
```

Comment:
Could this export cause an issue with another code executing this differently?

Reply:
That would be a good thing for me to investigate.

Reply:
Confirmed that this won't propagate to the calling shell unless someone uses `source <script>` to call it.